pages/common/mdn.md

```diff
@@ -0,0 +1,15 @@
+# mdn
+
+> Quick tool for search in Mozilla Developer Network from command line
```

Comment:
Is there a more information link we can include here too? Also, this will fix the linter error:
```suggestion
> Quick tool for search in Mozilla Developer Network from command line.
```